### DISCUSSION TITLE
Default import on Node.js doesn't return Alpine instance

### DISCUSSION THREAD

**Discussion by ypconstante** [alpinejs/alpine#3637] [7. Bugs (archived)]
I'm working on a project using Node test runner, and I'm trying to create tests for some modules we have today. 
When import Alpine, instead of getting an Alpine instance like when using a bundler, I get a `{default: Alpine}` object.

I'm using Node 18. 

Example:

package.json
```
{
  "type": "module",
  "dependencies": {
    "@happy-dom/global-registrator": "^9.20.3",
    "alpinejs": "^3.12.2"
  }
}
```

dom.js
````
import { GlobalRegistrator } from "@happy-dom/global-registrator";

GlobalRegistrator.register()
````

index.js
```
import "./dom.js"
import defaultExport from "alpinejs"
import { default as explicitDefaultExport } from "alpinejs"
import * as nameSpaceExport from "alpinejs"

console.log(defaultExport)
// output: { default: [Getter] }
console.log(explicitDefaultExport)
// output: { default: [Getter] }
console.log(nameSpaceExport)
// output: { default: { default: [Getter] } }
console.log(defaultExport.default.version)
// output: 3.12.2
```

The namespace output shows that there's an extra default wrapper that causes this issue.

**Comment by ekwoka** [comment]
This looks like you're not getting the actual esm version, but the cjs version.

Are you just running this explicitly or is it going through some preprocessing?

**Reply by ypconstante** [reply]
I just checked and indeed I'm getting the cjs version.
No preprocessing, just running `node index.js` with the files above is enough to reproduce.

**Reply by ekwoka** [reply]
Without any other direction, I believe Node defaults to the `main` field when no modern `exports` fields are present.

**Reply by ypconstante** [reply]
One way I found to fix this on Alpine side is to add `export {Alpine}` to `builds/module.js`, then I was able to import using `import { Alpine } from "alpinejs"`.
[This is the recommended solution from esbuild](https://esbuild.github.io/content-types/#default-interop).
